Assets/Scripts/MusicPlayer.cs

```diff
@@ -25,10 +30,13 @@ private void Awake()
 
     private void Start()
     {
-        _source = GetComponent<AudioSource>();        
-        _currentVolume = _source.volume;
-        PlayLoop(_source, _intro);
+        _musicSource = gameObject.AddComponent<AudioSource>();
+        _soundSource = gameObject.AddComponent<AudioSource>();
```

Comment:
We can avoid creation of `AudioSource` by just playing with `AudioSource.PlayClipAtPoint`.
`_musicSource` on the other hand should stay to allow `PlayLoop`. What benefits bring the dynamic creation instead of the one from Unity editor?

Reply:
Nice, changed